### DISCUSSION TITLE
Community space to share campaign files?

### DISCUSSION THREAD

**Discussion by theyellodart77** [kinkofer/FightClub5eXML#138] [Ideas]
I have Scoured the internet looking for a shared common space where DMs might be sharing not just the bestiaries or item lists but the loadable campaign files for various campaign settings.  I was able to find LMoP and CoS  but thats it.  Is there any  discussion about that being something that is added to the repository?

**Comment by zcdziura** [comment]
Nothing to my knowledge, but creating a repository here on GitHub to host such files for that purpose wouldn't be too difficult to set up.

**Reply by DaveT72** [reply]
Did this ever go anywhere? This repo seems focused on FightClub but would there be interest in extending it to include GameMaster and specifically campaigns?

**Comment by kinkofer** [comment]
The Game Master app uses the same format as Fight Club, so after you create
a Compendium it can be imported into either app.

**Reply by DaveT72** [reply]
Ya, imported the compendium I'm looking for campaigns that have been built.  Someone posted one for Lost Mine of Phandelver which was awesome.  Would love to be able to contribute content for the encounters, NPCs etc.  Unless I'm doing things wrong the building blocks seem to be in the compendium and we just need to figure out how to generate the campaigns from content that's available?

**Comment by theyellodart77** [comment]
This was my original hope.  But I never heard anything no. Sent from Mail for Windows From: DaveT72Sent: Monday, January 24, 2022 1:55 PMTo: kinkofer/FightClub5eXMLCc: theyellodart77; AuthorSubject: Re: [kinkofer/FightClub5eXML] Community space to share campaign files? (Discussion #138) Did this ever go anywhere? This repo seems focused on FightClub but would there be interest in extending it to include GameMaster and specifically campaigns?—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***>

**Comment by kinkofer** [comment]
Some of the newer sources haven't been added yet, but ones that released
prior to about 2020 are all there. There's some technical knowledge
required to build a compendium, but the instructions are available.
Building campaign encounters is outside the scope of this project.

**Reply by DaveT72** [reply]
ok fair enough, I can appreciate that.  All projects need to have a scope.  I'll keep looking for DM resources to define and share campaigns.

**Comment by leefujin** [comment]
If you do start new. I'd like to be involved
